libs/form-builder/src/lib/decorators/description-field.decorator.ts

```diff
@@ -20,6 +20,6 @@ export const DescriptionField = createFieldDecorator<
     label: 'description',
     formField: 'description',
     defaultValue: DefaultDescriptionValue(),
-    descriptionType: DescriptionType.HTML,
+    descriptionType: DescriptionType.HTML as DescriptionType, // otherwise aggregatefield type will treat it as html only,
```

Comment:
Not sure I understand the comment.

Reply:
to fix error regarding formField string type (i expected it to be actual 'description' | 'tags' etc, i made typescript imply type more strictly by using const in generic. Because of this descriptionType became DescriptionType.HTML instead of DescriptionType. Thats why i had to use as and specify actual type